bitbucket/resource_branching_model.go

```diff
@@ -21,11 +21,11 @@ type BranchingModel struct {
 }
 
 type BranchModel struct {
-	IsValid            bool   `json:"is_valid,omitempty"`
-	Name               string `json:"name,omitempty"`
-	UseMainbranch      bool   `json:"use_mainbranch,omitempty"`
-	BranchDoesNotExist bool   `json:"branch_does_not_exist,omitempty"`
-	Enabled            bool   `json:"enabled,omitempty"`
+	IsValid            bool    `json:"is_valid,omitempty"`
+	Name               *string `json:"name"`
```

Comment:
Changing this to a pointer allows for nullable types, where removing `omitempty` makes sure the `nil` value is not omitted when marshalling.